### PR TITLE
Replace crossbeam::mpmc with tokio's broadcast

### PR DESCRIPTION
**Pull request by future-highway**
Tokio's broadcast supports async/await

### CODE REVIEW

File: auraed/src/logging/logchannel.rs [hunk 1]
```diff
@@ -60,28 +59,24 @@ impl LogChannel {
...
-        match producer.send(LogItem {
+        // send returns an Err if there are no receivers. We ignore that.
+        let _ = producer.send(LogItem {
             channel: "unknown".to_string(),
             line: line.to_string(),
             // TODO: milliseconds type in protobuf requires 128bit type
             timestamp: get_timestamp_sec(),
-        }) {
-            Ok(_) => {}
-            Err(e) => {
```
Comment (dominichamon) [inline auraed/src/logging/logchannel.rs:75]: are we losing some error logging with this change?

Comment (future-highway) [reply auraed/src/logging/logchannel.rs:75]: Yes and no. LogChannel will only send messages while there is a receiver (connected gRPC client). Clients only get messages that are produced after they connect/subscribe. So if there is no receiver, LogChannel will lose messages.  However, SimpleLogger is also wired up, and that will receive all log messages I believe.

**Review by krisnova — APPROVED** [review]
Works for me

**Comment by future-highway** [comment]
I'm resisting the urge to press the merge button, to let Vincinator get a chance to review/comment.

But if anyone else wants to press the button, I won't object, lol.

**Review by krisnova — APPROVED** [review]

**Pull request merged**